prbt_support/test/acceptance_tests/acceptance_test_joint_limits.md

```diff
@@ -22,7 +22,7 @@ limitations under the License.
     skip the test for certain joints.
 
 ## Starting the joint limit acceptance tests
-  - Press the enabling switch and run `roslaunch prbt_support robot.launch`.
+  - Run `roslaunch prbt_support robot.launch`. Once the startup is complete press the acknowledge button.
```

Comment:
What's the "acknowledge button"? Is the test run in T1 or in Automatic mode?

Reply:
Thanks. I made an update.